.github/workflows/release.yml

```diff
@@ -0,0 +1,90 @@
+name: Create release
+on:
+  workflow_dispatch:
+    inputs:
+      release_type:
+        description: "The type of release (one of 'patch', 'minor', or 'major')"
+        required: true
+        default: patch
+        type: string
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Build
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+        with:
+          path: app-frontend
+          fetch-depth: 0 # Shallow clones should be disabled for a better relevancy of analysis
+
+      - name: install node
+        uses: actions/setup-node@v2
+        with:
+          node-version: '16'
+
+      - name: install dependencies
+        working-directory: app-frontend/src
+        run: yarn --immutable
+
+      - name: Prepare release version
+        working-directory: app-frontend/src/altinn-app-frontend
+        # This step runs if the trigger is run via github action
+        if: ${{ ! github.event.after }}
+        run: |
+          git config --global user.name 'github-actions[bot]'
+          git config --global user.email 'github-actions[bot]@users.noreply.github.com'
+          yarn release ${{ github.event.inputs.release_type }}
```

Comment:
I tried to find any documentation for this step, but failed.. Is it github workflow magic, or does it depend on a release script we don't currently have in package.json?

Reply:
Yeah, this is wrong - it should be `npm release`, not `yarn release`. Ill fix!

- [ ] fix incorrect command